packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -100,11 +100,16 @@ export class PivotBase extends React.Component<IPivotProps, IPivotState> {
     const divProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(this.props, divProperties);
 
     this._classNames = this._getClassNames(this.props);
+    const { renderActiveOnly } = this.props;
 
     return (
       <div role="toolbar" {...divProps}>
         {this._renderPivotLinks(linkCollection, selectedKey)}
-        {selectedKey && this._renderPivotItem(linkCollection, selectedKey)}
+        {renderActiveOnly !== false
+          ? selectedKey && this._renderPivotItem(linkCollection, selectedKey)
+          : linkCollection.links.map(link =>
+              this._renderPivotItem(linkCollection, link.itemKey, selectedKey === link.itemKey),
```

Comment:
Another option for this, instead of passing in a renderActiveOnly, you could delegate this to _renderPivotItem. So instead of having `renderActiveOnly` prop, each item might take a `renderPivotItem` prop which would have a signature like `renderPivotItem: (props: {index, isActive, className}) => JSX.Element`. That way it can handle a bunch more scenarios. It would allow more selective rendering as well, in the instance where only 1 should persist, or only 1 shouldn't persist.

Reply:
I agree, it definitely would make it more flexible, but looking at the feedback from the issue (and our own personal use case in Mission Control) it seems a bit overkill (YAGNI?) for what (IMO) should be provided out-of-the-box, + a lot more implementation from the consumer.